ConcurrentWitness2Test/start.sh svcomp23-witness-benchmarks/tools/infer/goblint-regression/28-race_reach_85-list2_racefree.yml/28-race_reach_85-list2_racefree.i --graphml-witness svcomp23-witness-benchmarks/tools/infer/goblint-regression/28-race_reach_85-list2_racefree.yml/witness.graphml


--------------------------------------------------------------------------------


/home/runner/work/ConcurrentWitness2Test/ConcurrentWitness2Test/ConcurrentWitness2Test/venv/lib/python3.12/site-packages/networkx/readwrite/graphml.py:1034: UserWarning: No key type for id creationtime. Using string
  warnings.warn(f"No key type for id {attr_id}. Using string")
Traceback (most recent call last):
  File "/home/runner/work/ConcurrentWitness2Test/ConcurrentWitness2Test/ConcurrentWitness2Test/main.py", line 84, in translate_to_c
    sys.exit(-1)
SystemExit: -1
/tmp/BenchExec_run_wezo_qa7/tmp/tmp8sc7x1xk.c
Compilation started
/tmp/BenchExec_run_wezo_qa7/tmp/tmp8sc7x1xk.c:715:8: error: redefinition of ‘struct s’
  715 | struct s
      |        ^
/tmp/BenchExec_run_wezo_qa7/tmp/tmp8sc7x1xk.c:710:8: note: originally defined here
  710 | struct s
      |        ^

Compilation ended (exit code 1)
Verdict: Compilation error
Verdict: Unknown error
